extension/js/common/ui/key-import-ui.ts

```diff
@@ -70,17 +72,47 @@ export class KeyImportUi {
     return;
   };
 
-  public static addAliasForSubmission = (email: string, submitKeyForAddrs: string[]) => {
-    submitKeyForAddrs.push(email);
+  // by unselecting, we allow to click on "Load from a file" and trigger the fineuploader again
+  public static allowReselect = () => {
+    $('input[type=radio][name=source]').prop('checked', false);
   };
 
-  public static removeAliasFromSubmission = (email: string, submitKeyForAddrs: string[]) => {
-    submitKeyForAddrs.splice(submitKeyForAddrs.indexOf(email), 1);
+  public renderEmailAliasView = async (acctEmail: string) => {
+    await Settings.refreshSendAs(acctEmail);
+    const storage = await AcctStore.get(acctEmail, ['email_provider']);
+    if (storage.email_provider === 'gmail') {
+      const { sendAs } = await AcctStore.get(acctEmail, ['sendAs']);
```

Comment:
what about merging 2 `AcctStore.get` into a single one?
and instead of putting code into the large `if (storage.email_provider === 'gmail') {`, we can just `return` if `email_provider` is not `gmail`:
```js
const { email_provider, sendAs } = await AcctStore.get(acctEmail, ['email_provider', 'sendAs']);
if (email_provider !== 'gmail') return;

const addresses = ...
```

Reply:
sounds good!